dashboard/error_handler.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, NamedTuple, Type
+
+from flask import (Blueprint, current_app, make_response, render_template,
+                   request)
+
+
+class ErrorHandler(NamedTuple):
+    exceptions: List[Type[Exception]]  # noqa
+    template_file: str
+    error_code: int
+
+
+def bind_error_handlers(blueprint: Blueprint,
+                        error_handlers: List[ErrorHandler]):
+    """
+    Wrapper of app_errorhandler for blueprints
+    to provide more comprehensive API
+    """
+    for handler in error_handlers:
+        for exception in handler.exceptions:
+            callback = _error_handler_callback(handler.template_file,
+                                               handler.error_code)
+            blueprint.app_errorhandler(exception)(callback)
+
+
+def _error_handler_callback(template, error_code):
+    def decorator(error):
+        current_app.logger.error(f'{error}; raised on URL: {request.url}')
```

Comment:
add error message
use `logger.exception`

Reply:
And also still save in log URL of request